lib/pfconfig/namespaces/FilterEngine/AccessScopes.pm

```diff
@@ -44,7 +50,7 @@ sub build {
             $logger->info("Building rule '$rule'");
             my ($parsed_conditions, $msg) = parse_condition_string($condition);
             unless (defined $parsed_conditions) {
-                warn("Error building rule '$rule' \n$msg");
+                $self->_error("Error building rule '$rule'", $msg);
```

Comment:
By not have the parse error message on it's own line.
The syntax error highlighting will be off.